README.md

```diff
@@ -1,31 +1,39 @@
 # Aldryn MailChimp
```

Comment:
That's our standard, but I don't think we should block on that, since this is already an improvement.